### ISSUE TITLE
1.19.2 Etherium Pickaxe is missing from forge:tools/pickaxes tag

### ISSUE THREAD

**Issue by ModernGamingWorld** [Extegral/Enigmatic-Legacy#424]
**Provide a detailed and clear description of feature or change that you want to offer.**
I would like to suggest you add your pickaxe to the forge:tools/pickaxes tag so that your pickaxe works with other mod related recipes that might call that tag in their recipes

**Describe why you want this. If there's a reason.**
So I and other modpack developers dont have to add it ourselves

**Comment by Toastr125** [comment]
And on a similar note, tagging the other tools and armor as well as the ingot, nugget, and block would be nice. I had to do that with a datapack for a modpack I've been making recently for compatibility with other mods.

**Comment by ModernGamingWorld** [comment]
@Toastr125 I agree I did not think about this, I just request this same feature to like 5 other developers as well that are not using tags for their pickaxes and never tested the armor or other tools, so nice suggestion

**Comment by Aizistral** [comment]
I approve this suggestion, but if I'm gonna do it then indeed, it makes sense to do it for all tools and armor. I will therefore have to look into available Forge tags to make sure I know and all appropriate ones. Since I don't want to delay ports because of this, I will do that in 1.20 releases.
